contracts/mapping-token/contracts/test/MockInboundLane.sol

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.6.0;
+import "./MockMessageVerifier.sol";
+import "@darwinia/contracts-bridge/contracts/interfaces/ICrossChainFilter.sol";
+import "hardhat/console.sol";
+
+contract MockInboundLane is MockMessageVerifier {
+    constructor(
+        uint32 _thisChainPosition,
+        uint32 _thisLanePosition,
+        uint32 _bridgedChainPosition,
+        uint32 _bridgedLanePosition
+    ) public MockMessageVerifier(
+        _thisChainPosition,
+        _thisLanePosition,
+        _bridgedChainPosition,
+        _bridgedLanePosition
+    ) {
+    }
+
+    function receive_message(address sender, address targetContract, bytes calldata encoded) external returns(bool) {
```

Comment:
This is miss leading and confuse with another method in InboundLane, should not be external in production.

```
function receive_message(Message[] memory messages) internal returns (uint256 dispatch_results) {
```